.github/workflows/build-pkg.yml

```diff
@@ -15,6 +15,9 @@ jobs:
       IDENTIFIER: sh.brew.Homebrew
       TMP_PATH: /tmp/brew
       MIN_OS: '11.0'
+      # Please replace with your team ID.
+      # https://developer.apple.com/help/account/manage-your-team/locate-your-team-id/
+      SIGNING_IDENTITY: '***'
```

Comment:
This should be a secret too right?

Reply:
Thanks for pointing out.
I think this ID isn't that secret because it'll be exposed in a common name of certificates.

Probably it's enough to make it as an environmental variable. (Not secret one.)
But I'm willing to change this as a secret.

This is a certificate of VirtualBox's installer. It contains `VB5E2TV963` as a common name.
<img width="480" alt="Screenshot 2023-02-20 at 10 48 39 PM" src="https://user-images.githubusercontent.com/1459450/220125487-a34c3125-b237-442a-a319-1a7422fed0b0.png">

Reply:
If it'll be public later just having it as a variable is enough.